thorlcr/activities/fetch/thfetchslave.cpp

```diff
@@ -43,10 +43,11 @@
 
 struct FPosTableEntryIFileIO : public FPosTableEntry
 {
-    FPosTableEntryIFileIO() { file = NULL; }
-    ~FPosTableEntryIFileIO() { ::Release(file); }
-    unsigned location;
```

Comment:
NB: not directly related to the issue, but noticed whilst reviewing changed.
'location' was uninitialized, but only ever used in an error report.